deployment/ansible/conf/jophiel.yml.j2

```diff
@@ -110,3 +110,7 @@ jophiel:
 
   superadminCreator:
     initialPassword: {{ jophiel_superadminCreator_initialPassword }}
+
+  session:
+    maxConcurrentSessionsPerUser: {{ jophiel_session_maxConcurrentSessionsPerUser }}
+    disableLogOut: {{ jophiel_session_disableLogOut }}
```

Comment:
When used as noun, it should be `logout`, not `logOut`. (so `disableLogout`).

Can we make the changes?